### DISCUSSION TITLE
How can I add line breaks or new line into the email?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16926] [How to] [answered]
Hello
I created a workflow which sends email about the overdue activities

However I can not figure out how can I add line breaks  to the email. If I hit enter, shift-enter, add table, etc,  the email will be always shrinked into one line. Never keep the formatting :(
What I am missing?

So this is what I saved
![image](https://user-images.githubusercontent.com/20862325/221350944-378caa4a-10e2-486f-8361-228cc556c137.png)

And after save it will be:
![image](https://user-images.githubusercontent.com/20862325/221350978-fa12193a-be5a-4da4-a159-9fa607e2bbb2.png)

And the email which i got:
![image](https://user-images.githubusercontent.com/20862325/221351069-52f06423-21ee-4442-8b4c-31dbc91bd421.png)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
use mail template + workflow
